src/pynxtools_raman/reader.py

```diff
@@ -79,6 +80,77 @@ def handle_eln_file(self, file_path: str) -> Dict[str, Any]:
 
         return {}
 
+
+    def parse_jdx_file(self, filepath):

```

Comment:
I suggest to move this to its own module (like the rod and witec readers) eventually.

Reply:
yes, was planned. Did not finish this yet.